### ISSUE TITLE
`ColliderBuilder` does not implement `Debug`

### ISSUE THREAD

**Issue by snyball** [dimforge/rapier#577]
The `ColliderBuilder` struct does not implement `Debug`, while it _is_ implemented for `RigidBodyBuilder`, this seems like an oversight.

**Comment by sebcrozet** [comment]
`ColliderBuilder` currently does not implement `Debug` mainly because it contains a trait-object for its shape. We could consider implementing `Debug` for `SharedShape` in `parry` by matching on know objects types though.

**Comment by whatf0xx** [comment]
Hi! I was looking to add a solution for this but I am confused by what you mean by 'matching on known objects types'. When I tried to look into doing a match on the interior of `SharedShape` in the `Debug` impl, I got errors, and upon looking into this on SO I found these related points:
https://stackoverflow.com/questions/26051433/how-do-you-match-a-trait-in-rust?rq=3 https://stackoverflow.com/questions/26126683/how-to-match-trait-implementors

Basically, I don't think you can do matching on the object types that implement `Shape`. Did I misunderstand what you were going for? Were you envisioning a solution with `Any` or something like that?

Sorry if these are dumb questions, I picked this thread indeed as a 'good first issue':)
